packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -31,6 +39,10 @@ - (FVPDisplayLink *)displayLinkWithRegistrar:(id<FlutterPluginRegistrar>)registr
 
 #pragma mark -
 
+/// The next non-texture player ID, initialized to a high number to avoid collisions with
+/// texture IDs (which are generated separately).
+static int64_t nextNonTexturePlayerId = 1000000;
```

Comment:
I'm not sure if this is the best approach, but looks like texture ids start from 0 and then get incremented, so this starting point seems to be pretty safe. Alternatively, I assume we could generate random ids for non-texture players

Reply:
could use `INFINITY`